docker/openproblems-python37-saucie/Dockerfile

```diff
@@ -0,0 +1,59 @@
+#FROM python:3.7
+FROM condaforge/mambaforge
+
+ENV DEBIAN_FRONTEND=noninteractive
+ARG NB_USER="sagemaker-user"
+ARG NB_UID="1000"
+ARG NB_GID="100"
+
+RUN \
+    apt-get update && \
+    apt-get install -qy sudo && \
+    useradd -m -s /bin/bash -N -u $NB_UID $NB_USER && \
+    chmod g+w /etc/passwd && \
+    echo "${NB_USER}    ALL=(ALL)    NOPASSWD:    ALL" >> /etc/sudoers && \
+    # Prevent apt-get cache from being persisted to this layer.
+    find /var/lib/apt/lists/ ! -type d -exec rm '{}' \;
+
+RUN apt-get update -qq
+RUN apt-get clean -y && apt-get autoremove -y && apt-get install -y cmake
```

Comment:
wouldn't you normally put clean and autoremove after all apt-get installations?

Reply:
fixed for docker containers in this PR now